src/index.html

```diff
@@ -17,11 +18,360 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
     />
   </head>
+  <!-- endregion -->
 
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo"
+      >
+        <img
+          src="../src/images/logo.png"
+          alt="Moyo Logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="is-active nav__link"
+            >
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              data-qa="nav-hover"
+              class="nav__link"
+            >
+              laptops & computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- #endregion -->
+    <main class="main">
+      <!-- #region card 1 -->
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="../src/images/imac.jpeg"
+          alt="imac"
+        />
+        <h1 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
```

Comment:
This tag used for title of the full page, consider will be better if you use the another tag here. Fix it everywhere
```suggestion
        <h2 class="card__title">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
        </h2>
```

Reply:
я не розуміюю. у мене це ж і є основним заголовком до карти. чи мається на увазі, що всі карти це другорядні на сторінці і може бути заголовок до блоку каталогу? @volodymyr-soltys97